pom.xml

```diff
@@ -53,6 +53,15 @@
       <artifactId>aws-java-sdk-sts</artifactId>
       <version>1.11.86</version>
     </dependency>
+    <!--
+      Older versions of jackson-databind have remote code execution vulnerabilities
+      CVEs - 2018-5968, 2017-17485, 2017-7525, 2017-15095, 2018-7489
```

Comment:
Can you mention why we can't just upgrade the aws version?

Reply:
Done.

Reply:
@manithirani 
Could you please help me how to use pom.xml
I have deployed the cloud watcher in my kubernetes , and I am getting some metrics like below , how to get those in prometheus 


Below is my config file

{
    "region": "us-west-2",
    "metrics": [
    {"aws_namespace": "AWS/ELB", "aws_metric_name": "HealthyHostCount",
     "aws_dimensions": ["us-west-2a", "test"], "aws_statistics": ["Average"]},
    {"aws_namespace": "AWS/ELB", "aws_metric_name": "UnHealthyHostCount",
     "aws_dimensions": ["us-west-2a", "test"], "aws_statistics": ["Average"]},
    {"aws_namespace": "AWS/ELB", "aws_metric_name": "RequestCount",
     "aws_dimensions": ["us-west-2a", "test"], "aws_statistics": ["Sum"]},
    {"aws_namespace": "AWS/ELB", "aws_metric_name": "Latency",
     "aws_dimensions": ["us-west-2a", "test"], "aws_statistics": ["Average"]},
    {"aws_namespace": "AWS/ELB", "aws_metric_name": "SurgeQueueLength",
     "aws_dimensions": ["us-west-2a", "test"], "aws_statistics": ["Maximum",    "Sum"]},
    ]
}

I can see below output

cloudwatch_requests_total 10.0

cloudwatch_exporter_scrape_duration_seconds 2.571412647

cloudwatch_exporter_scrape_error 0.0